src/Pages/AboutUsManager/AboutUsManagerModal.js

```diff
@@ -0,0 +1,285 @@
+        handleChange: e => setPictureUrl(e.target.value)
+      }
+    ]
+  ];
+
+  const inputMatrixOfficers = [
+    [
+      {
+        addon: 'First Name*',
+        type: 'text',
+        defaultValue: props.name,
+        placeholder: 'e.g. John D.',
+        maxlength: '24',
+        handleChange: e => setName(e.target.value)
+      },
+      {
+        addon: 'Email*',
+        type: 'email',
+        defaultValue: props.email,
+        handleChange: e => setEmail(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'LinkedIn Profile',
+        type: 'url',
+        placeholder:'Enter URL Here',
+        defaultValue: props.linkedin,
+        handleChange: e => setLinkedin(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'Profile Picture*',
+        type: 'url',
+        placeholder: 'Enter URL Here',
+        defaultValue: props.pictureUrl,
+        handleChange: e => setPictureUrl(e.target.value)
+      }
+    ]
+  ];
+
+  async function handleSubmission() {
+    const officerFields = {
+      name,
+      email,
+      linkedin,
+      team,
+      position,
+      quote,
+      pictureUrl
+    };
+    await props.handleSubmit({...officerFields});
+    window.location.reload();
+    props.toggle();
+  }
+
+  function requiredFieldsFilledIn() {
+    const nameEmailTeamPicture = name && email
+      && team && pictureUrl;
+    const linkedInQuotePosition = linkedin && quote && position;
+    if (team === 'executive' && (props.modalState === officerModalState.EDIT ||
+       props.modalState === officerModalState.SUBMIT)) {
+      return nameEmailTeamPicture && linkedInQuotePosition;
+    } else if (team === 'officers' &&
+       (props.modalState === officerModalState.SUBMIT ||
+       props.modalState === officerModalState.EDIT)) {
+      return nameEmailTeamPicture;
+    } return false;
+  }
+
+  return (
+    <div>
+      <ConfirmationModal {...confirmModalProps} />
+      <Modal isOpen={modal} size='lg' toggle={toggle}>
+        <ModalHeader>
+          {modalState === officerModalState.SUBMIT ? 'Create New ' : 'Edit '}
+          Officer
+        </ModalHeader>
+        <ModalBody>
+          <span>
+            <span color='red'>*</span>= This is a required field
+          </span>
+          <Row className='container'>
+            <Label>Team*</Label>
+            <Input
+              type='select'
+              defaultValue={props.team}
+              onChange={e => setTeam(e.target.value)}
+              children= {
+                <>
+                  <option>Select Team</option>
+                  <option value="officers">Officer Team</option>
+                  <option value="executive">Executive Team</option>
+                </>
+              }
+            />
+          </Row>
+          <br></br>
+          {team === 'executive'?
+            inputMatrixExec.map((row, index) => {
+              return (
+                <Row key={index} className='container'>
```

Comment:
`key={index}` is bad practice, we should really get a proper library to create React keys. For now, I think it's best to omit the key, unless you have a unique value you can use.